lib/topological_inventory/ansible_tower/operations/core/sources_api_client.rb

```diff
@@ -0,0 +1,85 @@
+require "sources-api-client"
+
+module TopologicalInventory
+  module AnsibleTower
+    module Operations
+      module Core
+        class SourcesApiClient < ::SourcesApiClient::ApiClient
+          INTERNAL_API_PATH = '//internal/v1.0'.freeze
+
+          def initialize(identity = nil)
+            super(::SourcesApiClient::Configuration.default)
+            self.identity = identity
+            self.api = init_default_api
+          end
```

Comment:
I get what you're doing here but subclassing a client gem here seems weird.  This is something that all our services currently do so if we're going to make it simpler lets do it either in the client gem or in a more common location

Reply:
ok. That was the easiest way to do thread-safe switch between public and internal api calls